orchestra/todos/views.py

```diff
@@ -190,13 +190,11 @@ def get_permissions(self):
         return [permission() for permission in permission_classes]
 
     def get_serializer_class(self):
-        if self.action == 'list' or self.action == 'create':
+        actions = ('list', 'create')
+        is_project_admin = self.request.user.groups.filter(
+            name='project_admins').exists()
+        if self.action in actions and is_project_admin:
             # Only include todo QA data for users in the
             # `project_admins` group.
-            if self.request.user.groups.filter(
-                    name='project_admins').exists():
-                return BulkTodoSerializerWithQA
-            else:
-                return BulkTodoSerializerWithoutQA
-        else:
-            return super().get_serializer_class()
+            return BulkTodoSerializerWithQA
+        return BulkTodoSerializerWithoutQA
```

Comment:
curiously, what happens to 
<img width="310" alt="image" src="https://user-images.githubusercontent.com/57725/92602964-c8fb4b80-f27c-11ea-885d-7b8d0ed5174f.png">